pkg/service/service.go

```diff
@@ -618,6 +628,7 @@ func (s *Service) createSVCInfoIfNotExist(p *lb.SVC) (*svcInfo, bool, bool,
 		((p.Type == lb.SVCTypeNodePort && !isWildcardAddr(svc.frontend)) ||
 			p.Type == lb.SVCTypeExternalIPs ||
 			p.Type == lb.SVCTypeLoadBalancer)
+	svc.maglev = svc.useMaglev()
```

Comment:
Is it necessary to set a boolean in this `svcInfo` struct?  Why not just call`useMaglev()`?

Reply:
It is a small optimization to avoid deriving whether Maglev should be used each time a service is updated or deleted. In any case, benefits of the optimizations are neglectable, so I'm going to replace reading of the bool field with the method invocation.